src/main/java/svydovets/util/ReflectionsUtil.java

```diff
@@ -41,10 +40,55 @@ public static List<String> findAutowiredFieldNames(Class<?> beanClass) {
 
         return Arrays.stream(fields)
                 .filter(field -> field.isAnnotationPresent(Autowired.class))
-                .map(Field::getName)
+                .map(field -> {
+                    if(field.isAnnotationPresent(Qualifier.class)) {
+                        Qualifier qualifier = field.getAnnotation(Qualifier.class);
+                        return qualifier.value();
+                    }
+
+                    return field.getName();
+                })
```

Comment:
We don't need this changes, because we have to know field name. For autowiring via field we will use smth like that: 
ComponentAnnotationBeanDefinition beanDefinition = (ComponentAnnotationBeanDefinition) beanDefinitionMap.get();
        Class<?> beanClass = beanDefinition.getBeanClass();
        beanDefinition.getAutowiredFieldNames().stream()
                .map(fieldName -> beanClass.getDeclaredField(fieldName))
                .forEach(field -> field.set(bean, getBean(field.getType())))

Reply:
Removed this changes